include/image.mk

```diff
@@ -589,12 +604,16 @@ endef
 
 define Device/DumpInfo
 Target-Profile: DEVICE_$(1)
-Target-Profile-Name: $(DEVICE_TITLE)
+Target-Profile-Name: $(DEVICE_TITLE)$(if $(strip $(DEVICE_ALT0_TITLE)), ($(DEVICE_ALT0_TITLE)))
```

Comment:
Why not semicolon-separate up to all three alternative names here? While comma might be used in DEVICE_VARIANT to list several variants, semicolon would be appropriate here imho.

Reply:
Isn't there barely enough space for two titles already?

Reply:
Just tried it with Putty from Windows here quickly, and there two titles will fill one line easily. Don't know whether it's the same directly on Linux.
I personally like the bracket solution `Vendor title (vendor title)` and would prefer it compared to `Vendor title; vendor title`, but I would not object if the latter was chosen.

Reply:
@dangowrt what do you think?